docs/pages/components/SkipLink.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: SkipLink
+description: The component is a user interface element designed to enhance website accessibility and usability. By clicking the skip link, users can efficiently navigate to the core content, improving their browsing experience and ensuring compliance with accessibility standards.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/SkipLink/index.tsx
+---
+
+import { SkipLink } from '@deque/cauldron-react'
+
+```jsx
+import { SkipLink } from '@deque/cauldron-react'
+```
+
+The skip link is at the very top of the page, but it’s “hidden”. The link will be “activated” by navigating through the page with the Tab button right after the page is loaded.
+
+## Example
+
+```jsx example
+<SkipLink target="#main-content" aria-label={"Skip"} />
```

Comment:
`aria-label` will override the internal labels, can we use the `skipText` and/or `targetText` props instead?